docs/content/contributing/bicep/bicep-contribution-flow/_index.md

```diff
@@ -100,55 +107,122 @@ For more examples, see the below script's parameters section.
-  - [Create a service principal (Azure Portal)](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
-  - [Create a service principal (PowerShell)](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-authenticate-service-principal-powershell)
-  - [Find Service Principal object ID](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/assign-roles-azure-service-principals#find-your-spn-and-tenant-id)
-  - [Find managed Identity Service Principal](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-view-managed-identity-service-principal-portal)
-- Note down the following pieces of information
-  - Application (Client) ID
-  - Service Principal Object ID (**not** the object ID of the application)
-  - Service Principal Secret (password)
-  - Tenant ID
-  - Subscription ID
-  - Parent Management Group ID
+{{< /expand >}}
 
 <br>
 
-## 2. Fork the module source repository
 
-{{< hint type=tip >}}
+## 2. Configure a deployment identity in Azure
 
-Checkout the [PowerShell Helper Script](#powershell-helper-script-to-setup-fork--ci-test-environment) that can do this step automatically for you! 👍
+AVM tests the deployments in an Azure subscription. To do so, it requires a deployment identity with access to it.
+
+{{< hint type=warning title="Deprecating the Service Principal + Secret authentication method">}}
+
+Support to Service Principal + Secret authentication method has been deprecated and will be decommissioned going forward.
+
+It is highly recommended to start leveraging Option 1 below to adopt OpenID Connect (OIDC) authentication and align with security best practices.
 
 {{< /hint >}}
 
-Bicep AVM Modules (both Resource and Pattern modules) will be homed in the [`Azure/bicep-registry-modules`](https://github.com/Azure/bicep-registry-modules) repository and live within an `avm` directory that will be located at the root of the repository, as per [SNFR19](/Azure-Verified-Modules/spec/SNFR19).
+{{< expand "➕ Option 1 [Recommended]: OIDC - Configure a federated identity credential" "expand/collapse" >}}
 
-Module owners are expected to fork the [`Azure/bicep-registry-modules`](https://github.com/Azure/bicep-registry-modules) repository and work on a branch from within their fork, before then creating a Pull Request (PR) back into the [`Azure/bicep-registry-modules`](https://github.com/Azure/bicep-registry-modules) repository's `main` branch.
+1. Create a new or leverage an existing user-assigned managed identity with at least `Contributor` & `User Access Administrator` permissions on the Management-Group/Subscription you want to test the modules in. You might find the following links useful:
```

Comment:
If wonder if it would make sense to recommend the newer [`Role Based Access Control Administrator`](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/privileged#role-based-access-control-administrator) role instead if the original [`User Access Administrator`](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/privileged#user-access-administrator).